UKHO.PeriodicOutputService/UKHO.BESS.CleanUpJob/settings.job

```diff
@@ -0,0 +1,3 @@
+{
+    "schedule": "0 15 16 * * *"
```

Comment:
This might need to revert to get the cleanup done in non working hours.

Reply:
This cron expression changed some time ago for dev testing purpose. Will revert it back to 0 0 3 * * * once dev testing is completed. Thanks!

Reply:
Ok